### ISSUE TITLE
Feedback about wallet tab / reduce options 

### ISSUE THREAD

**Issue by joseJRJR** [galacticcouncil/HydraDX-ui#788]
Tldr

Many complains about the "Buy" and "Sell" buttons, saying that these two buttons lead users to the same place, and at least one button can be removed.
I really miss the "Deposit" button, which would redirect to the cross-chain section.

So minimize the UI, and provide more clears buttoms
From here: https://hydradx.subsquare.io/posts/131

Some suggestion from the post are the next one 
![88946e736ea746c4ac9edae4b34278d5](https://github.com/galacticcouncil/HydraDX-ui/assets/123791232/72fed7d8-dff6-4c99-b57a-d6b49639f6d4)

Deposit" - should become a classic example of transferring tokens from one wallet to another!
"Transfer" - consists of a separate pop-up window featuring cross-chain messaging functionality!
We need similar mechanics as in regular classic exchanges.

Or 
Deposit" consists of a separate pop-up window featuring cross-chain messaging functionality. This means that users can transfer their tokens from their preferred network to the HydraDX network without needing to navigate to the cross-chain section. For example, if we choose to deposit Astar, the system will automatically select the source network as Astar.

Transfer" should become a classic example of transferring tokens from one wallet to another. Many users do not understand the purpose of this function, as they transfer tokens between wallets through browser wallets such as Polkadot JS, Talisman, etc. I propose to expand its capabilities and functionality, adding the ability to send funds to multiple wallets at once.

**Comment by jak-pan** [comment]
I think this is addressed on mainnet. Thanks for the feedback, if you have something else feel free to comment.
